src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebLogsRouteConfig.kt

```diff
@@ -0,0 +1,18 @@
+package org.vaccineimpact.orderlyweb.app_start.routing.web
+
+import org.vaccineimpact.orderlyweb.*
+import org.vaccineimpact.orderlyweb.app_start.RouteConfig
+import org.vaccineimpact.orderlyweb.controllers.web.LogsController
+
+object WebLogsRouteConfig : RouteConfig
+{
+    private val runReports = setOf("*/reports.run")
+
+    override val endpoints = listOf(
+            WebEndpoint("/running-reports/",
```

Comment:
We already have `/reports/runnable/` so would make sense for this to be `/reports/running/`?